recipes/wasmedge/all/conanfile.py

```diff
@@ -16,14 +16,15 @@ class WasmedgeConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/WasmEdge/WasmEdge/"
     topics = ("webassembly", "wasm", "wasi", "emscripten")
+    package_type = "shared-library"
     settings = "os", "arch", "compiler", "build_type"
 
     @property
     def _compiler_alias(self):
         return {
             "Visual Studio": "Visual Studio",
             "msvc": "Visual Studio",
-        }.get(str(self.settings.compiler), "gcc")
+        }.get(str(self.info.settings.compiler), "gcc")
 
     def configure(self):
         del self.settings.compiler.libcxx
```

Comment:
msvc won't like this and will fail for windows

Reply:
@RubenRBS 
Thanks a lot.
I fixed this problem.